File: C#/ 2984 Assuntos Pendentes.cs
using System;
using System.Collections.Generic;
using System.Linq;

class MainClass {
  public static void Main (string[] args) 
  {
    string s = Console.ReadLine();
    int fechamento = 0;
    for(int i = 0; i < s.Length; i++){
        if(s[i] == '(') fechamento++;
        else if(fechamento > 0) fechamento--;
    }
    if(fechamento <= 0) Console.WriteLine("Partiu RU!"); 
    else Console.WriteLine($"Ainda temos {fechamento} assunto(s) pendente(s)!");
    
  }
}

File: Python/2984 Assuntos Pendentes.py
s = input()
fechamento = 0
for i in range (len(s)):
    if s[i] == '(': fechamento+=1
    elif fechamento > 0: fechamento-=1
if fechamento <= 0: print("Partiu RU!")
else: print(f"Ainda temos {fechamento} assunto(s) pendente(s)!")